src/js/angular/externalsync/app.js

```diff
@@ -11,22 +11,9 @@ const modules = [
     'graphdb.framework.core.directives'
 ];
 
-const connectors = angular.module('graphdb.framework.externalsync', modules);
+const externalsync = angular.module('graphdb.framework.externalsync', modules);
```

Comment:
I maybe shouldn't have ranamed this as it was actually the connectors module and it was the package name which was incorrect I think. @desislava-hristova-ontotext

Reply:
But the module is externalsync now? I prefer externalsync actually, but this is not so important, so let's choose one and make it consistent.

Reply:
I see. OK, I'll keep it `externalsync` then.